apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobReadCarrierWithTaxNumbers.json

```diff
@@ -238,6 +238,28 @@
         "display" : "Primary provider"
       } ]
     }
+  }, {
+    "sequence" : 5,
+    "provider" : {
+      "identifier" : {
+        "type" : {
+          "coding" : [ {
+            "system" : "http://terminology.hl7.org/CodeSystem/v2-0203",
+            "code" : "TAX",
+            "display" : "Tax ID number"
+          } ]
+        },
+        "value" : "204299999"
+      }
+    },
+    "responsible" : true,
+    "role" : {
+      "coding" : [ {
+        "system" : "http://hl7.org/fhir/us/carin-bb/CodeSystem/C4BBClaimCareTeamRole",
+        "code" : "OTHER",
+        "display" : "Other"
+      } ]
+    }
```

Comment:
I think this is wrong, for both v2 and v1. Based on what Tawney said, the tax num is just yet another identifier for the performing physician. So we'd `identifier` here to be an array, containing entries for `CARR_PRFRNG_PIN_NUM`, `PRF_PHYSN_UPIN`, `PRF_PHYSN_NPI`, and `TAX_NUM` -- all in the same `careTeam` entry. Does that make sense?

I also _think_ that's a backwards compatible change for v1, though we should probably get Brady's and Bill's opinion on that.